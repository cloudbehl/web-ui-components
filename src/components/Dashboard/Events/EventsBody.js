import React from 'react';
import PropTypes from 'prop-types';

const EventsBody = ({ children, ...props }) => (
  <div {...props} className="kubevirt-events__body">
    {children}
  </div>
);

EventsBody.propTypes = {
  children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
};

export default EventsBody;
