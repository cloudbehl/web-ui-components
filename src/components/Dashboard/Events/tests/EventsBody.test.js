import React from 'react';
import { render } from 'enzyme';

import EventsBody from '../EventsBody';

import { default as EventsBodyFixture } from '../fixtures/EventsBody.fixture';

const testDetailsBody = () => <EventsBody {...EventsBodyFixture.props} />;

describe('<EventsBody />', () => {
  it('renders correctly', () => {
    const component = render(testDetailsBody());
    expect(component).toMatchSnapshot();
  });
});
