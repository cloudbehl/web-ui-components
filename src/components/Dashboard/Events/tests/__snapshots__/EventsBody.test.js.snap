// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EventsBody /> renders correctly 1`] = `
<div
  class="kubevirt-events__body"
  id="events-body"
>
  <div>
    Requires component from tectonic
  </div>
</div>
`;
